.claude/skills/osi-config-and-flags/SKILL.md

```
@@ -408,34 +414,41 @@ grep -n "osi-db-helper" conf/full_raspberrypi_bcm27xx_bcm2712/files/usr/share/no
+payload flip, the script issues `/etc/init.d/node-red restart` again and
+probes `/gui` before deciding whether to commit or auto-roll-back (rolling
+back restarts Node-RED a further time on the previous payload). The one
+deliberate exception is `migrate-cli.js` exit code `3` (backup-restore
+integrity failure) — that path leaves Node-RED stopped on purpose for
+operator intervention rather than restarting it. ChirpStack re-provisioning
+is unchanged: `osi-bootstrap` (`START=99`) only runs automatically on the
+next full boot, or can be triggered manually with
+`node /usr/share/node-red/chirpstack-bootstrap.js`.
 
 **Re-verify:**
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)